vscode/README.md

```diff
@@ -331,3 +331,17 @@ When `rubyLsp.formatter` is set to `auto`, Ruby LSP tries to determine which for
 
 If the bundle has a **direct** dependency on a supported formatter, such as `rubocop` or `syntax_tree`, that will be used.
 Otherwise, formatting will be disabled and you will need add one to the bundle.
+
+Here's a code example:
+
+```ruby
+# Gemfile
+group :development do
```

Comment:
Linters usually need to also run in CI, so let's say `group :development, :test do`. This matches what Rails 8 will do: https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/Gemfile.tt